git-cliff-core/src/error.rs

```diff
@@ -112,6 +112,10 @@ pub enum Error {
     /// See `require_conventional` option for more information.
     #[error("Requiring all commits be conventional but found {0} unconventional commits.")]
     UnconventionalCommitsError(i32),
+    /// Error raised when commits are not matched by any commit parser and the
+    /// `fail_on_unmatched_commit` option is enabled.
```

Comment:
Maybe link to the API docs like:

```suggestion
    /// [`GitConfig::fail_on_unmatched_commit`] option is enabled.
```

Reply:
applied